### PR TITLE
docs(purge-tool): update Purge Tool pages with latest improvements

### PR DESCRIPTION
**Pull request by akantcheff**
Update Purge Tool pages with the latest [release 2.1.0](https://github.com/bonitasoft/bonita-purge-tool/releases/tag/2.1.0)

Related to [BPM-304](https://bonitasoft.atlassian.net/browse/BPM-304)

[BPM-304]: https://bonitasoft.atlassian.net/browse/BPM-304?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/runtime/pages/purge-tool.adoc [hunk 1]
```diff
@@ -16,13 +17,15 @@ By default, all archives are preserved forever in Bonita runtime, but if your fu
 
 [WARNING]
 ====
-The purge tool doesn't delete documents (stored in the DOCUMENT table) from the platform. It will only remove the mapping between the archived cases and the document itself. If you need to reduce the size of the Document table in the engine database, please refer to the documentation: xref:ROOT:handling-documents.adoc#delete_document_archived_case[Delete documents of archived cases based on archive date]
+*Prior to version 2.1.0*, the Purge Tool did not delete documents (stored in the DOCUMENT table) from the platform. It only removed the mapping between the archived cases and the document itself. If you need to reduce the size of the Document table in the engine database, please refer to the documentation: xref:ROOT:handling-documents.adoc#delete_document_archived_case[Delete documents of archived cases based on archive date]
```
Comment (educhastenier) [inline modules/runtime/pages/purge-tool.adoc]:
```suggestion
*Prior to version 2.1.0*, the Purge Tool did not delete documents (stored in the DOCUMENT table) from the platform. It only removed the mapping between the archived cases and the document itself. If you need to reduce the size of the Document table in the engine database, consider using command `delete-orphan-document-content` to remove all documents not linked to any process instance anymore. Moreover, by default, subsequent runs of Purge Tool 2.1+ will automatically purge documents along with archived process instances, unless option `--preserve-document-content` is explicitly set.
```

File: modules/api/pages/handling-documents.adoc
Comment (educhastenier) [inline modules/api/pages/handling-documents.adoc:1]: Warning section of this page should also be updated to indicate that purge-tool (SP-only) can be used to delete documents.
And a new PR on 2025.1 should be done to indicate that documents are now deleted automatically when deleting an archived document.

File: modules/runtime/pages/purge-tool.adoc [hunk 2]
```diff
@@ -164,29 +179,41 @@ NOTE: The `--before-date` parameter must be in https://www.epochconverter.com/[m
...
-Due to the efficiency of the TRUNCATE command, this method should be faster in most cases, especially when the number of rows to delete is higher than the number of rows to keep in the table, but it requires the Runtime to be stopped during the operation.
+==== Fine-tuning the deletion
 
-Use the `--delete-mode copy-truncate` option to use this mode with the `delete` command.
+As mentioned before, both modes use batch processing to either delete or copy rows. +
+The batch processing can be fine-tuned using the `--batch-size` and `--batch-interval` options:
+
+    * `--batch-size` to define the number of rows to delete or copy in each batch. +
+With the `batch-delete` mode, although it depends on your database configuration and the volume of data to delete, we recommend starting with a batch size of `5000` rows. You can increase this value based on the performance of your database. +
+With the `copy-truncate` mode, you can use a larger batch size to improve performance.
```
Comment (educhastenier) [inline modules/runtime/pages/purge-tool.adoc]: How much larger? We should indicate a value, or a formula to calculate the value.

**Review by educhastenier — CHANGES_REQUESTED** [review]
RN 2025.1 should indicate the document deletion change

File: modules/api/pages/handling-documents.adoc
Comment (akantcheff) [reply modules/api/pages/handling-documents.adoc:1]: I was updating the page but I forgot :upside_down_face:

File: modules/runtime/pages/purge-tool.adoc (hunk 2 above)
Comment (akantcheff) [reply modules/runtime/pages/purge-tool.adoc]: To be honest, it is complicated to write a formula. I cannot be sure from when it is optimized and until when it is not.
We need to execute more performance tests with very large datasets (which we have not...)

**Comment by akantcheff** [comment]
> RN 2025.1 should indicate the document deletion change

Yes, it is planned with another PR

**Review by educhastenier — APPROVED** [review]

**Pull request merged**